### PR TITLE
fixes servo not work on speedybeef4

### PR DESCRIPTION
**Pull request by azolyoung**
HI there, 

This PR is fixed the servo not working on Speedy Bee F4. it has test by my coworker, every thing works fine!

### CODE REVIEW

File: src/main/target/SPEEDYBEEF4/target.c [hunk 1]
```diff
@@ -24,17 +24,18 @@
...
-    DEF_TIM(TIM8, CH3, PC8,  TIM_USE_MC_MOTOR                    | TIM_USE_FW_SERVO, 0, 1), // S3  UP(2,1)
-    DEF_TIM(TIM8, CH4, PC9,  TIM_USE_MC_MOTOR                    | TIM_USE_FW_SERVO, 0, 0), // S4  UP(2,1)
-    DEF_TIM(TIM2, CH1, PA15, TIM_USE_MC_MOTOR | TIM_USE_MC_SERVO | TIM_USE_FW_SERVO, 0, 0), // S5  UP(1,7)
-    DEF_TIM(TIM1, CH1, PA8,  TIM_USE_MC_MOTOR | TIM_USE_MC_SERVO | TIM_USE_FW_SERVO, 0, 0), // S6  UP(2,5)
+    DEF_TIM(TIM8, CH1, PC6,  TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR, 0, 0), // S1  UP(1,2)
+    DEF_TIM(TIM8, CH2, PC7,  TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR, 0, 1), // S2  UP(2,1)
 
-    DEF_TIM(TIM4, CH3, PB8,  TIM_USE_MC_MOTOR | TIM_USE_MC_SERVO | TIM_USE_FW_SERVO, 0, 0), // S7  D(1,7)!S5 UP(2,6)
-    DEF_TIM(TIM4, CH1, PB6,  TIM_USE_LED,                                            0, 0),   // LED D(1,0) UP(2,6)
+    DEF_TIM(TIM3, CH3, PC8,  TIM_USE_MC_SERVO | TIM_USE_FW_SERVO, 0, 0), // S3  UP(2,1)
```
Comment (DzikuVx) [inline src/main/target/SPEEDYBEEF4/target.c]: With those changes, Multirotor Configuration has not only 2 motors and thus it is not working!

**Review by DzikuVx — CHANGES_REQUESTED** [review]

File: src/main/target/SPEEDYBEEF4/target.c (hunk 1 above)
Comment (azolyoung) [reply src/main/target/SPEEDYBEEF4/target.c]: yeap! I forgot the quad needs support!

File: src/main/target/SPEEDYBEEF4/target.c [hunk 2]
```diff
@@ -24,17 +24,18 @@
...
 
-    DEF_TIM(TIM8, CH1, PC6,  TIM_USE_MC_MOTOR                    | TIM_USE_FW_MOTOR, 0, 0), // S1  UP(1,2)
-    DEF_TIM(TIM8, CH2, PC7,  TIM_USE_MC_MOTOR                    | TIM_USE_FW_MOTOR, 0, 1), // S2  UP(2,1)
-    DEF_TIM(TIM8, CH3, PC8,  TIM_USE_MC_MOTOR                    | TIM_USE_FW_SERVO, 0, 1), // S3  UP(2,1)
-    DEF_TIM(TIM8, CH4, PC9,  TIM_USE_MC_MOTOR                    | TIM_USE_FW_SERVO, 0, 0), // S4  UP(2,1)
-    DEF_TIM(TIM2, CH1, PA15, TIM_USE_MC_MOTOR | TIM_USE_MC_SERVO | TIM_USE_FW_SERVO, 0, 0), // S5  UP(1,7)
-    DEF_TIM(TIM1, CH1, PA8,  TIM_USE_MC_MOTOR | TIM_USE_MC_SERVO | TIM_USE_FW_SERVO, 0, 0), // S6  UP(2,5)
+    DEF_TIM(TIM8, CH1, PC6,  TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR, 0, 0), // S1  UP(1,2)
+    DEF_TIM(TIM8, CH2, PC7,  TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR, 0, 1), // S2  UP(2,1)
+    DEF_TIM(TIM3, CH3, PC8,  TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR, 0, 0), // S3  UP(2,1)
```
Comment (DzikuVx) [inline src/main/target/SPEEDYBEEF4/target.c]: @azolyoung alsmost, but then FW needs only 2 motors, not 4. S3 and S4 can be assigned as FW servos

**Review by DzikuVx — CHANGES_REQUESTED** [review]

File: src/main/target/SPEEDYBEEF4/target.c (hunk 2 above)
Comment (azolyoung) [reply src/main/target/SPEEDYBEEF4/target.c]: Got it, thanks!

**Review by DzikuVx — APPROVED** [review]

**Pull request merged**